curriculum/challenges/english/25-front-end-development/lab-factorial-calculator/66c07238b01053abaf812065.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 66c07238b01053abaf812065
+title: Build a Factorial Calculator
+challengeType: 14
+dashedName: build-a-factorial-calculator
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should ask the user for a number using a prompt window and store the number in a variable called `inputNumber`.
+   
+2. Declare a variable `result` and assign it the value of `1`.
+   
+3. Using a for loop, you should loop through all numbers from `1` to `inputNumber` (inclusive) and for each number, multiply the `result` variable by the current number and assign the result to the `result` variable.
```

Comment:
we should allow for any type of loop here.
we should only care that they use some sort of loop and get the correct final result

```suggestion
3. Using a loop, loop through all numbers from `1` to `inputNumber` (inclusive) and for each number, multiply the `result` variable by the current number and assign the result to the `result` variable. You can choose to use  either a`for` loop, `while` loop or `do...while` loop here.
```